packages/api-server/index.js

```diff
@@ -19,12 +20,13 @@ app.use(cookieParser());
 
 //route
 app.use('/api/auth', authRoute);
+app.use('/api', accountRoute);
 
 app.use((err, req, res) => {
```

Comment:
this middleware is used to catch internal server error. if you guys want this to response a specific error, let's find a better way (use `Error` object).